Gems/Atom/RPI/Code/Include/Atom/RPI.Edit/Common/JsonUtils.h

```diff
@@ -119,6 +123,59 @@ namespace AZ
                 return false;
             }
 
+            using DeserializedCpuData = AZStd::vector<RHI::CpuProfilingStatisticsSerializer::CpuProfilingStatisticsSerializerEntry>;
```

Comment:
Question: any thoughts on this function's placement? There's also a `JsonUtils.h` in AtomCore but since we need many of the Atom types I opted to include it here.